sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsClient.java

```diff
@@ -454,6 +454,16 @@ public void deleteRelationship(String digitalTwinId, String relationshipId) {
     // Models APIs
     //==================================================================================================================================================
 
+    /**
+     * Creates one or many models.
+     * @param models The list of models to create. Each string corresponds to exactly one model.
+     * @return A {@link PagedIterable} of created models and the http response.
+     */
+    @ServiceMethod(returns = ReturnType.COLLECTION)
+    public PagedIterable<ModelData> createModels(List<String> models) {
+        return new PagedIterable<>(digitalTwinsAsyncClient.createModels(models, Context.NONE));
+    }
```

Comment:
Not sure if it was deliberate, but the sync client didn't have the minimal API for creating models, so I added it. @azabbasi can you comment on if we need this API or not?

Reply:
As @azabbasi confirmed, we will need this minimal sync API

Reply:
correct, this API is now added in master.